packages/ui/components/select-rich/src/LionSelectRich.js

```diff
@@ -85,6 +85,15 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
     );
   }
 
+  /**
+   * @protected
+   * @configure FocusMixin
+   */
+  // @ts-ignore
+  get _focusableNode() {
```

Comment:
Why `get _focusableNode` when the element is named `_invokerNode`?

Reply:
The `_focusNode` is used inside the `FocusMixin`. For all inputs this corresponds with the `_inputNode`, only in case of the SelectRich it has to be the `_invokerNode`, since that is the element that will get focus.